modules/model/shared/src/main/scala/observe/model/StepState.scala

```diff
@@ -31,10 +31,10 @@ object StepState {
     }
 
   implicit class StepStateOps(val s: StepState) extends AnyVal {
-    def canSetBreakpoint(i: Int, firstRunnable: Int): Boolean = s match {
+    def canSetBreakpoint: Boolean = s match {
       case StepState.Pending | StepState.Skipped | StepState.Paused | StepState.Running |
           StepState.Aborted =>
-        i > firstRunnable
+        true
```

Comment:
Is it so that you can always set a breakpoint?

Reply:
No, I changed the logic a bit, because the `i > firstRunnable` part does not make sense with `StepId`. So now the function in `StepState` only checks that the state is right for setting a breakpoint. The caller then checks that the step is after the `firstRunnable` in the list order.